docs/zh/latest/plugins/limit-count.md

```diff
@@ -42,8 +42,8 @@ title: limit-count
 | ------------------- | ------- | --------------------------------- | ------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | count               | integer | 必须                               |               | count > 0                                                                                               | 指定时间窗口内的请求数量阈值                                                                                                                                                                                                                                                                                                                                                                                          |
 | time_window         | integer | 必须                               |               | time_window > 0                                                                                         | 时间窗口的大小（以秒为单位），超过这个时间就会重置                                                                                                                                                                                                                                                                                                                                                                    |
-| key_type      | string | 可选   |  "var"      | ["var", "var_combination"]                                          | key 的类型 |
-| key           | string  | 可选   |    "remote_addr"    |  | 用来做请求计数的依据。如果 `key_type` 为 "var"，那么 key 会被当作变量名称。如果 `key_type` 为 "var_combination"，那么 key 会当作变量组。比如如果设置 "$remote_addr $consumer_name" 作为 keys，那么插件会同时受 remote_addr 和 consumer_name 两个 key 的约束。如果 key 的值为空，$remote_addr 会被作为默认 key。 |
+| key_type      | string | 可选   |  "var"      | ["var", "var_combination", "constant"]                                          | key 的类型 |
+| key           | string  | 可选   |    "remote_addr"    |  | 用来做请求计数的依据。如果 `key_type` 为 “constant”，那么 key 会被当作常量。如果 `key_type` 为 "var"，那么 key 会被当作变量名称。如果 `key_type` 为 "var_combination"，那么 key 会当作变量组。比如如果设置 "$remote_addr $consumer_name" 作为 key，那么插件会同时受 remote_addr 和 consumer_name 两个变量的约束。如果 key 的值为空，$remote_addr 会被作为默认 key。 |
```

Comment:
`“constant”` -> `"constant"`?

Reply:
Better to avoid mixing English punctuation with Chinese doc

Reply:
yes, but `“` is Chinese punctuation